### DISCUSSION TITLE
Footnotes in markdown

### DISCUSSION THREAD

**Discussion by diepoe** [pngwn/MDsveX#223] [Q&A] [answered]
Does MDSvex support footnotes in markdown like described [here](https://www.markdownguide.org/extended-syntax#footnotes)? Because if I try to use this syntax, it doesn't seem to work...

**Comment by pngwn** [comment]
Is the link you posted correct?

**Comment by pngwn** [comment, marked as answer]
Footnotes are not part of the markdown spec and aren't supported by mdsvex by default, you can use any remark plugin to extend the markdown syntax though (for the time being).

The [`remark-footnotes`](https://github.com/remarkjs/remark-footnotes) plugin would do the trick here, make sure you use version 2. Add it to your mdsvex configuration [as described here](https://mdsvex.com/docs#remarkplugins--rehypeplugins) and it should work just fine.

**Reply by diepoe** [reply]
Ok thank you very much for your help, works fine now 🚀
